### PR TITLE
Fix broken spec, update gems.

### PR DESCRIPTION
**Pull request by eschlange**
* Updated TFD Engine and Dashboard gems.
* Fixed the reports spec.

[Finished #89640832]

### CODE REVIEW

File: spec/features/reports_spec.rb [hunk 1]
```diff
@@ -31,8 +31,8 @@
     click_on "Module Session"
 
     expect(page.body).to start_with(
-      "participant_id,module_id,page_headers,module_selected_at," \
-      "last_page_number_opened,last_page_opened_at\n"
+      "participant_id,module_id,page_headers,module_selected_at,"\
+      "last_page_opened_at\n"
```
Comment (michaelwehrley) [inline spec/features/reports_spec.rb]: @ericcf Weren't you working on these changes too?  If so, could you double check?  Thanks!
:+1: :haircut:

Comment (ericcf) [inline spec/features/reports_spec.rb]: Yes, this was one of those issues where my change in tfd engine broke this spec in both host apps :-(. Sorry.

Comment (michaelwehrley) [inline spec/features/reports_spec.rb]: @ericcf okay, so the changes in reports_spec are correct?

**Comment by ericcf** [comment]
Roger that.

**Pull request merged**